src/goto-instrument/accelerate/overflow_instrumenter.cpp

```diff
@@ -211,38 +212,25 @@ void overflow_instrumentert::overflow_expr(
           expr.id()==ID_mult)
   {
     // A generic arithmetic operation.
-    multi_ary_exprt overflow(
-      "overflow-" + expr.id_string(), expr.operands(), bool_typet());
-
-    if(expr.operands().size()>=3)
+    // The overflow checks are binary.
+    for(std::size_t i = 1; i < expr.operands().size(); i++)
```

Comment:
This is not a request for this PR but more a general observation that this code seems to be not covered by the regression tests.  I have a feeling that at one point Matt had tests for this code.  Maybe @cristina-david could help?

Reply:
We have regression/acceleration, it just seems the script in there needs some love to make this work. Adding this to my to-do list.